### PR TITLE
4735 log reducing

### LINKED ISSUE #4735: Reduce excessive error logging in JUnit Tests
The Travis build log is _polluted_ with debug logging lines. They _almost certainly_ related to some maths work Ian was working on related to detecting when a ship turns.  

They were inserted so that Ian could look at the performance on 'real' user data, using a deployed build of Debrief.  But, that venture was suspended.

So, they really should be commented out.  It's a case of doing `search-files` for some identifier in the error listing.

### PR DESCRIPTION
**Pull request by saulhidalgoaular**
Fix #4735

### CODE REVIEW

File: org.mwc.cmap.legacy/src/MWC/GUI/Editable.java [hunk 1]
```diff
@@ -397,8 +397,8 @@ public static void testTheseParameters(final Editable toBeTested)
           final int len = pd.length;
           if (len == 0)
           {
-            System.out.println("zero property editors found for " + toBeTested
```
Comment (IanMayo) [inline org.mwc.cmap.legacy/src/MWC/GUI/Editable.java]: @saulhidalgoaular  the error logging in this file is needed - it helps us track badly configured editors (such as when we don't provide the getter/setter for a named property)

So, we should keep the `system.out` entries in this file.

File: org.mwc.debrief.track_shift/src/org/mwc/debrief/track_shift/ambiguity/AmbiguityResolver.java [hunk 2]
```diff
@@ -1462,7 +1462,7 @@ else if (valueDelta > 180)
                 + " lastBrg:" + (int) lastCut.getBearing() + " brg:"
                 + ((int) cut.getBearing()) + " brg delta:" + brgRate
                 + " gap rate:" + gapRate + " OS zig trip:" + TRIP_ZIG;
-            doLog(logger, stats, time);
```
Comment (IanMayo) [inline org.mwc.debrief.track_shift/src/org/mwc/debrief/track_shift/ambiguity/AmbiguityResolver.java]: @saulhidalgoaular - in this file, a _slightly_ less intrusive change would be to just comment out the contents of `doLog()`.

**Comment by IanMayo** [comment]
Thanks @saulhidalgoaular  - all good, just two minor changes, please.

**Comment by saulhidalgoaular** [comment]
> Thanks @saulhidalgoaular - all good, just two minor changes, please.

Changes Applied :+1:

**Pull request merged**